src/services/WorkPeriodService.js

```diff
@@ -273,9 +273,11 @@ async function updateWorkPeriod (currentUser, id, data) {
   if (_.isNil(thisWeek)) {
     throw new errors.ConflictError('Work Period dates are not compatible with Resource Booking dates')
   }
+  /* https://github.com/topcoder-platform/taas-apis/issues/428
   if (thisWeek.daysWorked < data.daysWorked) {
     throw new errors.BadRequestError(`Maximum allowed daysWorked is (${thisWeek.daysWorked})`)
   }
+  */
```

Comment:
As we allow up to `10` and don't care about RB.endDate anymore, let's remove this code.